phpBB/config/default/container/services_files.yml

```diff
@@ -1,4 +1,35 @@
 services:
+    attachment.delete:
```

Comment:
these services should be moved to somewhere else (as they are crashing the install env).